CleverPush/Source/CleverPush.h

```diff
@@ -150,6 +150,7 @@ extern NSString * const CLEVERPUSH_SDK_VERSION;
 + (NSString*)getSubscriptionId;
 + (NSString*)getApiEndpoint;
 + (NSString*)channelId;
++ (NSString*)getAppBannerId;
```

Comment:
what is "appBannerId" - should we rather do "currentAppBannerId" ?

Reply:
it `( appBannerId )` was a variable name for getting the bannerId. I will change it to `currentAppBannerId` from `appBannerId`